app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -3,14 +3,46 @@
   var dom = window.shared.ReactHelpers.dom;
   var createEl = window.shared.ReactHelpers.createEl;
   var merge = window.shared.ReactHelpers.merge;
-
+  var FeedHelpers = window.shared.FeedHelpers;
   var QuadConverter = window.shared.QuadConverter;
 
+  var Colors = {
+    math: '#e8e9fc',
+    reading: '#ffe7d6',
+    procedural: '#e8fce8'
+  };
+  var styles = {
+    feedCard: {
+      marginBottom: '10px',
```

Comment:
In React styles, you can use a plain number and it will always interpret that as a pixel value.  And for font sizes, let's use pixels instead of points.  I'll add a doc about this, since it's come up a few times in the last few weeks.